testData/solutions/win32.json5

```diff
@@ -7572,7 +7643,9 @@
             "settings.configurePointerPrecision",
             "settings.configureSnapToDefButton",
             "settings.configureHidePointer",
-            "settings.configureMouseSonar"
+            "settings.configureMouseSonar",
+            "settings.configureDoubleClickWidth",
```

Comment:
Shouldn't these two additions go into the restore block too?

Reply:
Indeed they should, nice catch, changing it.